### ISSUE TITLE
Competitive job warning is too sensitive

### ISSUE THREAD

**Issue by NickHeiner** [rowanmanning/joblint#78]
I don't think that the phrase "Build on top of our platform." should trigger a competitiveness warning, but it does.

```
 λ curl --data "{\"spec\":\"Build on top of our platform.\"}" --header "Content-Type:application/json" --silent http://joblint.org/ws | json
{
  "errors": [],
  "warnings": [],
  "notices": [
    {
      "message": "The job sounds competitive and performance-based",
      "detail": "Competition can be healthy, but for a lot of people a heavily competitive environment can be a strain. You could also potentially be excluding people who have more important outside-of-work commitments, such as a family.",
      "evidence": [
        "top"
      ]
    }
  ],
  "failPoints": {
    "culture": 0,
    "realism": 1,
    "recruiter": 1,
    "tech": 0
  }
}
```

**Comment by rowanmanning** [comment]
Makes sense, "top" is too broad. I'll see if I can fix
